src/components/Drawer/Drawer.tsx

```diff
@@ -78,13 +78,6 @@ export const Drawer = ({
   const ref = useRef<HTMLDivElement | null>(null);
   const isMountedRef = useRef(true);
 
-  /**
-   * Use effect
-   * 1) Set prevIsActive to previous isActive prop
-   * 2) If prevIsActive is defined and previous isActive prop is not equal
-   * to current isActive prop, toggle state
-   */
```

Comment:
Maybe leave this block just to reduce merge conflicts with the other PR addressing comments.